src/main/java/io/quarkus/gizmo/ClassCreator.java

```diff
@@ -85,6 +85,14 @@ public ClassCreator(ClassOutput classOutput, String name, String signature, Stri
         this(null, classOutput, name, signature, superClass, ACC_PUBLIC | ACC_SUPER | ACC_SYNTHETIC, interfaces);
     }
 
+    public MethodCreator getConstructorAsMethodCreator(String... parameters) {
```

Comment:
Seems fine to me but I would not call the method this as it implies you are getting a constructor rather than a method builder. Maybe `getConstructorMethodBuilder` instead?

Reply:
Renamed.

Reply:
Hm, I would personally use something like `getConstructorCreator()`. It's not ideal but aligns with thet current "wording". Also `MethodCreator` is not a real builder...

Reply:
Agree, it should be called `getConstructorCreator()`.